tests/unit/r2i/tflite/loader.cc

```diff
@@ -0,0 +1,112 @@
+/* Copyright (C) 2018 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+*/
+
+#include <r2i/r2i.h>
+#include <r2i/tflite/loader.h>
+#include <fstream>
+#include <memory>
+
+#include <CppUTest/CommandLineTestRunner.h>
+#include <CppUTest/MemoryLeakDetectorNewMacros.h>
+#include <CppUTest/MemoryLeakDetectorMallocMacros.h>
+#include <CppUTest/TestHarness.h>
+
+static bool incompatible_model = false;
+static bool pass_fake_model = false;
+
+r2i::tflite::Model::Model () {
+}
+
+r2i::RuntimeError r2i::tflite::Model::Start (const std::string &name) {
+  return r2i::RuntimeError();
+}
+
+r2i::RuntimeError r2i::tflite::Model::Set (
+  std::shared_ptr<::tflite::FlatBufferModel>
+  tfltmodel) {
+  r2i::RuntimeError error;
+
+  if (incompatible_model) {
+    error.Set (r2i::RuntimeError::Code::INCOMPATIBLE_MODEL, "Incompatible model");
+  }
+
+  return error;
+}
+
+TEST_GROUP (TensorflowliteLoader) {
+  r2i::RuntimeError error;
+  std::shared_ptr<r2i::tflite::Loader> loader{new r2i::tflite::Loader};
+  std::string path;
+};
+
+TEST (TensorflowliteLoader, EmptyString) {
+  std::string empty_string = "";
+
+  /* Attempt to load this file as a valid model */
+  auto model = loader->Load(empty_string, error);
```

Comment:
Watch out! you are using the same error for all the tests. You **must** clean it for every test. Create a `setup()` function to do this.